www/config.yml

```diff
@@ -326,3 +326,12 @@ chapters:
       twitter: pyladiesdc
       meetup: dc-pyladies
 
+    - name: Dallas, TX
```

Comment:
For some reason, you have `+`s on every line.

Also, please keep this in alphabetical order.  Dallas should be placed after Chicago.